run_xnat2bids.py

```diff
@@ -6,52 +7,105 @@
 import subprocess
 from toml import load
 
-# Load config file into dictionary
-arglist = load('x2b_config.toml')
-
-# Extract arglist from dictionary
-for key in arglist:
-
-    # Populate sbatch arguments
-    if (key == "sbatch-args"):
-        time = arglist[key]['time'] 
-        mem = arglist[key]['mem'] 
-        nodes = arglist[key]['nodes'] 
-        cpus = arglist[key]['cpus'] 
-        job =  arglist[key]['job'] 
-        output = arglist[key]['output'] 
-        email = arglist[key]['email']
-
-    # Populate xnat2bids arguments
-    if(key == "xnat2bids-args"):
-        version = arglist[key]['version'] 
-        data_dir = arglist[key]['data_dir'] 
-        bids_root = arglist[key]['bids_root'] 
-        session = arglist[key]['session'] 
+def extractParams(param, value):
+    arg = []
+    for v in value:
+        arg.append(f"--{param} {v}")
+    return ' '.join(arg)
+
+
+def compileArgumentList(arg_dict):
```

Comment:
Maybe it feels overkilled, but it totally feels like we should have a test for this function. What do you think?

Reply:
An not the interactive user/password part but t the dict parsing